projects/angular/src/forms/combobox/combobox.ts

```diff
@@ -257,14 +257,19 @@ export class ClrCombobox<T>
   }
 
   onBlur() {
-    this.onTouchedCallback();
-    if (this.control.control.updateOn === 'change' && this.control.control?.errors?.required) {
+    this.focused = false;
+
+    if (this.onTouchedCallback) {
+      this.onTouchedCallback();
+    }
```

Comment:
```suggestion
    this.onTouchedCallback?.();
```

You can simplify this.

Reply:
But why do we need the conditional call? The previous code had an unconditional call.

Reply:
The function is assigned dynamically and in some cases is `undefined` or `null` .

Reply:
corrected in commit 254c211